misc/package/Data Files/MWSE/core/mwse/tes3/getKeyName/main.lua

```diff
@@ -0,0 +1,13 @@
+---@param keyCode tes3.scanCode
+function tes3.getKeyName(keyCode)
+    -- get the index of the appropriate GMST setting
+    -- the `%02X` does hexadecimal formatting and gives numbers `< 16` a leading 0.
+    --      so, `11` -> "0A" and `16` -> "10"
+    -- for example, if `keyCode == tes3.scanCode["2"] == 3`, then
+    --      `string.format("sKeyName_%02X", keyCode) == sKeyName_03`
+	local index = tes3.gmst[string.format("sKeyName_%02X", keyCode)]
+    -- if the game setting exists, return its value
+	if index then
+		return tes3.findGMST(index).value
+	end
```

Comment:
Let's change the formatting a bit: please use tabs for indentation and turn comments into proper sentences.

Reply:
Thanks for mentioning the spaces being used instead of tabs. I don't even know how the spaces snuck into this one. I also updated the comments so they're written in full sentences.